mammoth/models/model_saver.py

```diff
@@ -95,12 +96,14 @@ def save(self, step, moving_average=None):
                 self._rm_checkpoint(todel)
             self.checkpoint_queue.append(chkpt_names)
 
-    def _save(self, step):
+    def _save(self, step, save_model, data_state, device_context):
         """Save a resumable checkpoint.
 
         Args:
             step (int): step number
             model (nn.Module): torch model to save
```

Comment:
missing: save_model (type): description

Reply:
yeah, i think it was renamed from model to save_model? not sure why. will fix that.